_topics/taproot.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Taproot
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Taproot** is a proposed soft fork change to Bitcoin that will allow
+  public keys to commit to a complex script that can be revealed at
+  spend time.
+
+## Optional.  Use Markdown formatting.  Multiple pargraphs.  Links allowed.
+extended_summary: |
+  Coins protected by taproot may be spent either by satisfying the
+  complex script or by simply providing a signature that verifies
+  against the public key (allowing the script to be kept private).
+  Taproot is intended for use with schnorr signatures that simplify
+  multiparty construction (e.g. using [MuSig][]) and with MAST to
+  allow the pruning of unused script conditions.
```

Comment:
I prefer to talk about 'multiple scripts' and 'pruning unused scripts' rather than 'a complex scripts' and 'pruning unused script conditions'. That seems to map more intuitively to the idea of tapscript and that each script can potentially use different script versions or languages.